features/signals_railway_signals.yaml

```diff
@@ -1012,6 +1019,202 @@ features:
+    tags:
+      - { tag: 'railway:signal:train_protection', value: 'AU:NSW:end_signalled_authority' }
+      - { tag: 'railway:signal:train_protection:form', value: 'sign' }
+
+  - description: End Single Light Indication
+    country: AU
+    icon: { default: 'au/nsw/signs/end_single_light' }
+    tags:
+      - { tag: 'railway:signal:train_protection', value: 'AU:NSW:end_single_light' }
+      - { tag: 'railway:signal:train_protection:form', value: 'sign' }
+
+  - description: End Train Order Working
+    country: AU
+    icon: { default: 'au/nsw/signs/end_TOW' }
+    tags:
+      - { tag: 'railway:signal:train_protection', value: 'AU:NSW:end_TOW' }
+      - { tag: 'railway:signal:train_protection:form', value: 'sign' }
+
+  - description: End Yard Limit
+    country: AU
+    icon: { default: 'au/nsw/signs/EYL' }
+    tags:
+      - { tag: 'railway:signal:departure', value: 'AU:NSW:EYL' }
+      - { tag: 'railway:signal:departure:form', value: 'sign' }
+
+  - description: Landmark
+    country: AU
+    icon: { default: 'au/nsw/signs/landmark' }
+    tags:
+      - { tag: 'railway:signal:station_distant', value: 'AU:NSW:landmark' }
+      - { tag: 'railway:signal:station_distant:form', value: 'sign' }
+
+  - description: Loading Gauge Limit
+    country: AU
+    icon: { default: 'au/nsw/signs/loading_gauge' }
+    tags:
+      - { tag: 'railway:signal:stop', value: 'AU:NSW:loading_gauge' }
+      - { tag: 'railway:signal:stop:form', value: 'sign' }
+
+  - description: Location
+    country: AU
+    icon: { default: 'au/nsw/signs/location' }
+    tags:
+      - { tag: 'railway:signal:station_distant', value: 'AU:NSW:location' }
+      - { tag: 'railway:signal:station_distant:form', value: 'sign' }
+
+  - description: No Whistle
+    country: AU
+    icon: { default: 'au/nsw/signs/no_whistle' }
+    tags:
+      - { tag: 'railway:signal:whistle', value: 'AU:NSW:no_whistle' }
+      - { tag: 'railway:signal:whistle:form', value: 'sign' }
+
+  - description: Points Cleared
+    country: AU
+    icon: { default: 'au/nsw/signs/points_cleared' }
+    tags:
+      - { tag: 'railway:signal:fouling_point', value: 'AU:NSW:points_cleared' }
+
+  - description: Shunting Limit
+    country: AU
+    icon: { default: 'au/nsw/signs/shunting_limit' }
+    tags:
+      - { tag: 'railway:signal:shunting', value: 'AU:NSW:shunting_limit' }
+      - { tag: 'railway:signal:shunting:form', value: 'sign' }
+
+  - description: Signalised Level Crossing
+    country: AU
+    icon: { default: 'au/nsw/signs/signalised' }
+    tags:
+      - { tag: 'railway:signal:crossing_hint', value: 'AU:NSW:signalised' }
+      - { tag: 'railway:signal:crossing_hint:form', value: 'sign' }
+
+  - description: Stop Position
+    country: AU
+    icon:
+      match: 'railway:signal:stop:states'
+      cases:
+        - { any: ['X', 'XPT', 'XPL'], value: 'au/nsw/signs/stop_position/X' }
+        - { any: ['10'], value: 'au/nsw/signs/stop_position/10' }
```

Comment:
Exact matches can use `exact` (for example `exact: '10'`), although this will also work.

See for example https://github.com/k-yle/OpenRailwayMap-vector/blob/9225057d9c3a9754c2f15b09232c68a5d12f9e02/features/signals_railway_signals.yaml#L358